sections/1-introduction.tex

```diff
@@ -1,101 +1,76 @@
 \end{itemize}
 
 As of the time of writing, there are a total of 8 SEPs. 
-Some notables SEPs have lead to the adoption of physical units throughout the code base (SEP-0003, see Section~\ref{sec:units}), defined the affiliated package program (SEP-0004, see Section~\ref{sec:affil_package}), standardized the use of coordinate and coordinate transformations (SEP-0005, see Section~\ref{sec:coords}), and led to the adoption of a high precision scientific time format (SEP-0008).
+Some notable SEPs led to the adoption of physical units throughout the code base (SEP-0003, see Section~\ref{sec:units}), defined the affiliated package program (SEP-0004, see Section~\ref{sec:affil_package}), standardized the use of coordinate and coordinate transformations (SEP-0005, see Section~\ref{sec:coords}), and led to the adoption of a high precision scientific time format (SEP-0008).
 
 \section{Support \& Sustainability}
-% edited by Steven C. 23-Apr-2019
 
-The \sunpyproj relies largely on unpaid, volunteer efforts from early-career scientists which donate their time sometimes as part of their regular work duties.
-The \sunpy project has not received any significant direct financial support for its work facilitating and promoting open-source and open development software including developing the \sunpypkg. 
+The \sunpyproj and relies largely on unpaid, volunteer efforts from early career scientists. 
+The project has not received any significant direct financial support for its work facilitating and promoting any open-source and open development software including \sunpy itself. 
+The \astropy community faces a similar problem \citep{PriceWhelan:2018ji, Muna2016}. 
 
-Some development was funded by the Google Summer of Code (GSOC) and the ESA Summer of Code in Space (SOCIS) as well as a small grant from NumFOCUS. 
-\todo{sdc - point to a list of all students, isn't this available on sunpy.org?}
-\todo{Not sure if its full list but we have this: https://github.com/sunpy/sunpy/wiki/Wall-of-Fame}
+The SunPy Project does participate in two summer of code programs, which offer students stipends to write code for open-source projects, via the OpenAstronomy umbrella organization. Since 2013, 16 students contributed to SunPy through the Google Summer of Code (GSoC) program. Since 2011, 7 students contributed to SunPy through a similar program funded by the European Space Agency called The ESA Summer of Code in Space. Many SunPy community members served as mentors for these students\footnote{\url{https://github.com/sunpy/sunpy/wiki/Wall-of-Fame}}. In 2018, NumFOCUS, a 501(c)(3) public charity which collects and manages tax-deductible contributions for many open-source scientific software packages such as NumPy and AstroPy, gave GSoC student Vishnunarayan K. I. an award for exceptional contributions to the open source scientific software ecosystem for his efforts to update SunPy with a modern astronomical time system.
```

Comment:
Link to the pages for GSOC and SOCIS here?

Reply:
Done.